src/witan/app/handler.clj

```diff
@@ -160,15 +160,16 @@
                                (forecast/forecast {:id id :version version}))
                    (sweet/POST* "/data" {:as request}
                                 :multipart-params [file :- upload/TempFileUpload
+                                                   filename :- String
                                                    name :- String
                                                    category :- String
-                                                   public :- Boolean]
+                                                   public? :- Boolean]
                                 :middlewares [wrap-multipart-params]
                                 :summary "Upload,validate and save a data file"
                                 (data/data {:category category
                                             :name name
                                             :file file
-                                            :public public
+                                            :public public?
```

Comment:
I only put this back in to keep it consistent with other instances of `public?` throughout the app.